### ISSUE TITLE
/system/identities/issue doesn't works

### ISSUE THREAD

**Issue by brian02oriel** [hyperledger/composer#4672]
## Context
I'm using the HL Explorer Rest Server tu create new Identities. When i did it manually i only introduce something like:

-"participant": "<participant name>",
 -"userID": "<resource>#<ID>",

and i get an Internal Server Error (500) from the server with the message "Missing namespace".
How can i do this, when i try to run in the HL Explorer it just keep loading for a long time.

**Comment by nimit199** [comment]
```
"participant": ""
"userID": "#",
```

Needs to be
```
"participant": "org.my-network.participantModelName.Participant",
"userID": "#RandomString(preferablyNumbers)"
```
```org.my-network.participantModelName.Participant ``` Tells you what kind of participant you want to create and ```"userID": "#"``` is thier ""identified by" ID

If you want to create a new you need to have resource access of ```org.hyperledger.composer.system.AddParticipant```
to add a participant,
```org.hyperledger.composer.system.IssueIdentity```
to issue them identity and
```org.hyperledger.composer.system.ActivateCurrentIdentity```
to acticate the identity.

If you want to create an entirely new participant(not defined in model file) then you need to add it to your model file and upgrade your network to the new version.

Let me know if this solves your issue

**Comment by ryjones** [comment]
⚠️ ⚠️ ⚠️

As of the 29th August 2019, the Hyperledger Composer project is in deprecated status. None of the maintainers are actively developing new features. None of the maintainers are actively providing support via GitHub issues. However, if you wish to submit code changes via pull requests, these will be merged.

It is highly recommended that you use Hyperledger Fabric v1.4+ instead, which features significant improvements to the developer experience, including a new programming model.

More information available here: What's new in Hyperledger Fabric v1.4

⚠️ ⚠️ ⚠️
